Page 1 add reactive dropdown

cjrace opened the pull request:
## Pull request overview

Attempt to add a region dropdown onto the first page and then make the chart and table selections update that... Not ready to review yet as still very much a work in progress, will pick this up again later in the week.

## Pull request checklist

Please check if your PR fulfills the following:
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been reviewed and added / updated if needed (for bug fixes / features)
- [ ] Tests have been run locally and are passing (`shinytest2::test_app()`)
- [ ] Code is styled according to tidyverse styling (checked locally with `styler::style_dir()` and `lintr::lint_dir()`)

## What is the current behaviour?

...

## What is the new behaviour?

...

## Anything else

...

cjrace committed:
WIP - added dropdown but not connected properly

cjrace committed:
WIP - added dropdown but not connected properly

cjrace committed:
regenerate manifest

Merge branch 'page-1-add-reactive-dropdown' of https://github.com/dfe-analytical-services/apprenticeships-provider-dashboard into page-1-add-reactive-dropdown

# Conflicts:
#	manifest.json

cjrace committed:
Still WIP - not yet connected the dropdown fully. Left todo markers for next steps

axcooper committed:
region dropdwon now observes and is updated by selecting a region bar in the chart

axcooper committed:
removed commented out code

johalastrahol reviewed (approved):
All looks fine now. Region bar highlighted as expected

axcooper committed:
working on region bars

axcooper committed:
better punctuation than underscore in region dropdown

axcooper committed:
fixed typo

axcooper committed:
removed some commented out ocde

axcooper committed:
made dropdowns go along top

axcooper committed:
working on updating region / home table from region dropdown

axcooper reviewed (approved):
all ok

axcooper committed:
sorting conflicts

Merge branch 'main' of https://github.com/dfe-analytical-services/apprenticeships-provider-dashboard into page-1-add-reactive-dropdown

# Conflicts:
#	R/dashboard_modules/01-provider_breakdowns.R
#	data/subjects_and_standards_0.parquet
#	manifest.json

axcooper committed:
ensured input boxes are correct width and removed instruction

axcooper committed:
uncommenting code for filtering providers

axcooper committed:
note on selections

axcooper committed:
think have got the dropdown to work anbd everything to talk to each other

axcooper committed:
think have got the dropdown to work anbd everything to talk to each other

axcooper committed:
think working

Merge branch 'page-1-add-reactive-dropdown' of https://github.com/dfe-analytical-services/apprenticeships-provider-dashboard into page-1-add-reactive-dropdown

# Conflicts:
#	R/dashboard_modules/01-provider_breakdowns.R

axcooper committed:
learner home region in table selection sorted

axcooper committed:
amended legth of selected providers

axcooper committed:
select = integer(0) rather than NULL

axcooper committed:
put label back in to bar chart on page 1
